### ISSUE TITLE
automove not work

### ISSUE THREAD

**Issue by saburovga** [hadouken/hadouken#151]
Automove don't work. My hadouken config:

{
    "bittorrent":
    {
        "defaultSavePath": "E:/Torrent/Temp",
        "statePath": "C:/ProgramData/Hadouken//state/",

        "anonymousMode": false,

        "dht":
        {
            "enabled": true,
            "routers":
            [
                [ "router.bittorrent.com",  6881 ],
                [ "router.utorrent.com",    6881 ],
                [ "dht.transmissionbt.com", 6881 ],
                [ "dht.aelitis.com",        6881 ]
            ]
        },

        "proxy":
        {
            "type": "none",
            "hostname": "",
            "port": 0,
            "username": "",
            "password": "",
            "proxyHostnames": true,
            "proxyPeerConnections": true
        },

        "storage":
        {
            "sparse": true
        },

        "upnp":
        {
            "enabled": true
        }
    },

    "extensions":
    {
        "autoadd":
        {
            "enabled": true,
			"folders":
			[
				{ 
					"path": "E:/Torrent/Uploads",
					"tags": [ "Auto", "Add" ]
				}
			]
        },

        "automove":
        {
            "enabled":  true,
			"rules":
			[
				{
					"path": "E:/Torrent/Downloads",
					"filter": "tags",
					"data": [ "Auto", "Add" ]
				}
			]
        },

        "launcher":
        {
            "enabled": false
        },

        "pushbullet":
        {
            "enabled": false,
            "token": "YOUR-PUSHBULLET-AUTH-TOKEN",
            "enabledEvents":
            [
                "hadouken.loaded",
                "torrent.added",
                "torrent.finished"
            ]
        }
    },

    "http":
    {
        "address": "0.0.0.0",
        "port": 7070,

        "auth":
        {
            "type": "basic",
            "token": "",
            "basic":
            {
                "userName": "user",
                "password": "Pass"
            }
        },

        "root": "/",

        "ssl":
        {
            "enabled": false,
            "privateKeyFile": "",
            "privateKeyPassword": ""
        }
    },

    "scripting":
    {
        "path": "C:/ProgramData/Hadouken//js"
    }
}

.torrent file added to torrent list and downloaded successfully. But don't move into "Downloads" folder after download finished.

My hadouken version is 5.0.0

**Comment by vktr** [comment]
Hi, this is a bug in the `automove.js` file. Sorry for this. I'll push a fix for this as soon as possible. Here's how you can fix it righ away,

* Open `C:/ProgramData/Hadouken/js/plugins/automove.js` in a text editor.
* Change line 30 from

  ```
  var torrentTags = torrent.metadata.get("tags");
  ```
  to
  ```
  var torrentTags = torrent.metadata("tags");
  ```

* Save the file and restart Hadouken.

**Comment by saburovga** [comment]
Hi, Viktor. I fix line 30 in file automove.js, but it's don't solve my problem.

**Comment by vktr** [comment]
Hi, I fixed the JS issue in #156 (the same way you fixed yours) and tested it using the same `autoadd` and `automove` configuration as you. This is an excerpt of my configuration,

```
"extensions":
{
  "autoadd":
  {
    "enabled": true,
    "folders": [
      {
        "path": "C:/tmp/Torrents",
        "tags": [ "Auto", "Add" ]
      }
    ]
  },
  "automove":
  {
    "enabled": true,
    "rules": [
      {
        "path": "C:/Downloads/AutoMoveTarget",
        "filter": "tags",
        "data": [ "Auto", "Add" ]
      }
    ]
  }
}
```

All torrents I put in `C:\tmp\Torrents` will be automatically moved to `C:\Downloads\AutoMoveTarget` when finished.
